src/components/flight-views/flight-synchronized-charts.jsx

```diff
@@ -80,7 +81,7 @@ export default class FlightSynchronizedCharts extends React.Component {
       renderTo: 'launch-distance-chart',
       title: 'Distance from launch',
       tooltipPointFormatter: function() {
-        return `<span>${this.y} ${pilotDistanceUnit}</span>`;
+        return pilotDistanceUnit == 'm' ? `<span>${Math.round(this.y)} ${pilotDistanceUnit}</span>` : `<span>${this.y.toFixed(3)} ${pilotDistanceUnit}</span>`;
       },
```

Comment:
I would suggest to add a method to `Distance` util that will accept distance in kilometers and return a formatted string, either `n km` or `n mi` depending on pilot's selected unit. And the same method can check whether a value in pilot units is small enough to format it as `n m` or `n ft`.

Something like this:

```javascript
smallUnits: {
  kilometers: {
    factor: 1000,
    unit: ‘meter’,
    unitShort: ‘m’
  },
  {
    factor: 5280,
    unit: ‘feet’,
    unitShort: ‘ft’
  },
};

formatDistance(distance) {
  const pilotDistanceUnit = this.getUserDistanceUnit();
  let distanceInPilotUnit = this.getDistanceInPilotUnits(distance);
  let pilotDistanceUnitShort = this.getUserDistanceUnitShort();
  if (distanceInPilotUnit < 1) {
    distanceInPilotUnit = Math.round(distanceInPilotUnit * this.smallUnits[pilotDistanceUnit].factor);
    pilotDistanceUnitShort = this.smallUnits[pilotDistanceUnit].unitShort;
  }
  return `${distanceInPilotUnit} ${pilotDistanceUnitShort}`;
}
```

Reply:
OK, keep that for later, I am not too attached to meters anyway :-)